File: TFTesterJava/JarTester/src/main/java/com/davincium/jartester/MainActivity.java
package com.davincium.jartester;

import android.os.Bundle;
import android.app.Activity;
import android.util.Log;
import android.view.Menu;
import android.view.View;

import com.testflightapp.lib.TestFlight;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


    }

    public void clickedButton(View v)
    {
        Log.d("MainActivity", "Clicked button");
        TestFlight.log("Clicked Button");
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.main, menu);
        return true;
    }
    
}

File: TFLib/TFTester/MainActivity.cs
using System;
using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;

using MonoDroid.TestFlightLib;

namespace TFTester
{
	[Application (Name = "com.davincium.tftester.TFTester.AppDelegate")]			
	public class AppDelegate : Application
	{
		public override void OnCreate ()
		{
			base.OnCreate ();

			TestFlight.TakeOff (this,"some-kind-of-uid");
		}

		#region MonoDroid Hacks
		// MonoDroid Hacks for a Sub-Classed Application
		//public AppDelegate(IntPtr handle) : base (handle){}
		protected AppDelegate(IntPtr a,JniHandleOwnership b):base(a,b){} 

		#endregion
	}

	[Activity (Label = "JarTester", MainLauncher = true)]
	public class MainActivity : Activity
	{
		int count = 1;

		protected override void OnCreate (Bundle bundle)
		{
			base.OnCreate (bundle);



			// Set our view from the "main" layout resource
			SetContentView (Resource.Layout.Main);

			// Get our button from the layout resource,
			// and attach an event to it
			Button button = FindViewById<Button> (Resource.Id.myButton);
			
			button.Click += delegate
			{
				TestFlight.LogEvent("Clicked Button");
				button.Text = string.Format ("{0} clicks!", count++);
			};
		}
	}
}
